yazenfiras/style.css

```diff
@@ -0,0 +1,111 @@
+*{
+box-sizing: border-box;
+margin: 0%;
+padding: 0%;
+}
+h3{color: white;}
+
+.aside{
+position: fixed;
+    height: 100vh;
+    width: 18%;
+
+    background-color: #2F4F4F;
+    display: inline-block;
+    float: left;
+text-align: right;
+padding-top: 10%;
+font-family: Arial, Helvetica, sans-serif;
+padding-right: 1%;
+
+
+
+
+}
+.a1{
+
+    border-style: inset;
+    padding: 2%;
+    display: inline-block;
+    border-color:white;
+    text-decoration: none;
+    margin: 2%;
+    color:white;
+    border-width: 1px;
+}
+.p1{color: white;}
+
+.main{display: block;
+
+    margin-left:18.5%; 
+    padding-top:2%;
+    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+    font-style: italic;
+    
+
+font-size: 18px;
+
+}
+
+.avatar{
+    width:40px;
+    height: 40px;
+
+    float: right;
+    margin-right: 10px;
+    margin-top:0%;
+    border-radius: 50%;
+    margin-bottom: 10px;
+  }
+  .css{color:white;
+background-color: green;
+border:inset;
+border-width: 1px;
+padding: 2px;
+padding-left: 8px;
+padding-right: 8px;}
+
+.js{color:white;
+    background-color:blue;
+    border:inset;
+    border-width: 1px;
+    padding: 2px;
+    padding-left: 8px;
+    padding-right: 8px;}
+    
+    .js2{color:white;
+        background-color:red;
+        border:inset;
+        border-width: 1px;
+        padding: 2px;
+        padding-left: 8px;
+        padding-right: 8px;}
+
+
+        .not{color:white;
+            background-color:#2F4F4F;
+            border:inset;
+            border-width: 1px;
+            padding: 2px;
+            padding-left: 6px;
+            padding-right: 6px;}
+        
+            .img{width:47%;
+                height: 47%;
+            display: inline-block;
+            }
+            .img2{width:44%;
+                height: 47%;
+            display: inline-block;
+            }
+            .footer{
+                padding-left:33%;
+                display: inline-block;
+            }
+        
```

Comment:
For the spaces
Don't leave too many empty lines and the end.
also, try to add one empty line between each rule.